neptune/api_exceptions.py

```diff
@@ -21,12 +21,55 @@ class NeptuneApiException(NeptuneException):
     pass
 
 
-class SSLError(NeptuneException):
+class NeptuneSSLVerificationError(NeptuneException):
     def __init__(self):
-        super(SSLError, self).__init__(
-            "SSL certificate validation failed. Set NEPTUNE_ALLOW_SELF_SIGNED_CERTIFICATE "
-            "environment variable to accept self-signed certificates."
-        )
+        message = """
+{h1}
+----NeptuneSSLVerificationError-----------------------------------------------------------------------
+{end}
+
+Neptune client was unable to verify your SSL Certificate.
+
+{bold}What could go wrong?{end}
+    - You are behind a proxy that inspects traffic to Neptune servers.
+        - Contact your network administrator
+    - Your on-prem installation's SSL/TLS certificate is not recognized due to a custom Certificate Authority (CA).
+        - To check run the following command in terminal:
+            {bash}curl https://<your_domain>/api/backend/echo {end}
+        - Where <your_domain> is the address that you use to access Neptune UI i.e. abc.com
+        - Contact your network administrator if you get the following output:
+            {fail}"curl: (60) server certificate verification failed..."{end}
+    - Your machine software is not up-to-date.
+        - Minimal OS requirements:
+            - Windows >= XP SP3
+            - macOS >= 10.12.1
+            - Ubuntu >= 12.04
+            - Debian >= 8
+
+{bold}What can I do?{end}
+You can manually configure Neptune to skip all SLL checks. To do that
```

Comment:
`SLL` -> `SSL`

Reply:
Thanks, it is fixed ✅

Reply:
it's not in the copy-pasted message of the second exception ;)

Reply:
Can you elaborate?

Reply:
No, it's not!

I did a complete search for SLL on all scripts just to double check.

Reply:
😅 It was, but I fixed it ✅ 

vscode global search couldn't match 'SLL' the second file 😞